experiment/api.js

```diff
@@ -0,0 +1,21 @@
+
+const { classes: Cc, interfaces: Ci, utils: Cu } = Components;
+
+
+this.tabsplit = class API extends ExtensionAPI {
+  getAPI(context) {
+    return {
+      tabsplit: {
+        init: async () => {
```

Comment:
I think the API should have some clearer names. Like `increaseSidebarMaxWidth` or something

Reply:
Good idea.